rcsbapi/data/schema.py

```diff
@@ -15,12 +16,40 @@
 except ImportError:
     use_networkx = True
 
-PDB_URL: str = "https://data.rcsb.org/graphql"
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```
In addition to this change, can you also add the following lines to the `__init__.py` file? This should make the logging cleaner and not coming from "root":
```python
import logging

logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s]-%(module)s.%(funcName)s: %(message)s")
logger = logging.getLogger()
logger.setLevel(logging.INFO)
```